prow/github/client.go

```diff
@@ -3243,7 +3246,26 @@ func (c *client) CloseIssue(org, repo string, number int) error {
 		method:      http.MethodPatch,
 		path:        fmt.Sprintf("/repos/%s/%s/issues/%d", org, repo, number),
 		org:         org,
-		requestBody: map[string]string{"state": "closed"},
+		requestBody: map[string]string{"state": "closed", "state_reason": "completed"},
```

Comment:
The default value of `state_reason` is `completed` if nothing is passed, but I thought passing it explicitly might be better because while testing we won't be able to test for `state_reason` when using `CloseIssue` since that is not populated by default in the client.

Reply:
+1, explicit is better than implicit.